### PR TITLE
Dynamic Content Display : https://github.com/cross-solution/cross-solution.de/issues/26

### PR DESCRIPTION
**Pull request by Hetal2425**
=> Component managed
=> Dynamic content displayed on the right hand side
=> HTML tags converted to HTML view.

### CODE REVIEW

File: src/pages/post-a-job.vue [hunk 1]
```diff
@@ -306,27 +387,23 @@ export default {
...
-    titleQualifications: String,
-    benefits: String,
-    titleBenefits: String,
-    titleContact: String,
-    contact: String
+    }
   },
   mounted () {},
   methods: {
+    myfun () {},
```
Comment (cbleek) [inline src/pages/post-a-job.vue]: seems unnecessary. If so, please remove

File: src/pages/post-a-job.vue [hunk 2]
```diff
@@ -286,7 +368,6 @@
 <script lang="javascript">
 import YSearchRegion from '../components/SearchRegion.vue'
 import YAddress from '../components/Form/Address.vue'
-import YJob from '../components/Form/Job.vue'
```
Comment (cbleek) [inline src/pages/post-a-job.vue]: Why you've removed the component?

**Comment by Hetal2425** [comment]
Moved q-splitter section to Job component. So directly get the variable values there.

If we need to share the variable between the components then also we can use the emit method to share the values.I have added that code in files.

**Comment by cbleek** [comment]
it works

**Pull request merged**